_1327/information_pages/forms.py

```diff
@@ -7,5 +12,30 @@ class Meta(DocumentForm.Meta):
 		model = InformationDocument
 		fields = ['title', 'text', 'url_title', 'is_menu_page', 'comment']
 
+	def __init__(self, *args, **kwargs):
+		user = kwargs.pop('user', None)
+		creation = kwargs.pop('creation', None)
+		super().__init__(*args, **kwargs)
+		staff = Group.objects.get(name=settings.STAFF_GROUP_NAME)
+
+		if user.is_superuser:
+			permitted_groups = Group.objects.filter(permissions__codename="add_informationdocument")
```

Comment:
the only difference I can see, is in the filtering of the groups, but maybe I'm wrong :sweat_smile: